cmd/wallet-web/src/pages/demos/Connections.vue

```diff
@@ -1,50 +1,49 @@
-/*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
+/* Copyright SecureKey Technologies Inc. All Rights Reserved. SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Can we exclude these changes ?

Reply:
Yes, the easiest solution would be to replace the comment style with the following:
```
<!--
Copyright SecureKey Technologies Inc. All Rights Reserved.

SPDX-License-Identifier: Apache-2.0
-->
```
This way these lines won't be linted into single line.

@rolsonquadras